translationtable/GLOBAL_TERMS.yaml

```diff
@@ -485,7 +486,6 @@
 "ends_with": "RO:0002230",
 "has_input": "RO:0002233",
 "colocalizes with": "RO:0002325",
-"contributes to": "RO:0002326",
```

Comment:
Let's double check if this affects any other sources, this was used occasionally before "contributes to condition" was added to RO.  Possibly used in GWAS, OMIM

Reply:
Added "contributes to": "RO:0002326" back at least for now